server/bench-wrk/hge_wrk_bench.py

```diff
@@ -259,6 +266,7 @@ def max_rps_test(self, query):
             user = self.get_current_user()
         )
         summary = json.loads(result)['summary']
+        # TODO explain this calculation. Why aren't we using wrk's reported 'max'? Should we call this avg_sustained_rps or something?
```

Comment:
This is the average service rate of graphql-engine. `max` and `min` are just the extremes of the service rate distribution. Those are not useful if we want to compute the maximum allowed requests/sec for `wrk2` tests.